workflow-controller/controller-service/src/test/resources/servicespecs/test1.hspec.yaml

```diff
@@ -0,0 +1,12 @@
+name: test1
```

Comment:
I think all the resources should named to relevant names.
Can you re-think on the naming convention for all resources